src/EnergyPlus/PlantLoopEquip.cc

```diff
@@ -326,7 +326,7 @@ namespace PlantLoopEquip {
 				}
 
 			} else if ( EquipTypeNum == TypeOf_Chiller_Absorption ) {
-				SimBLASTAbsorber( sim_component.TypeOf, sim_component.Name, EquipFlowCtrl, LoopNum, LoopSideNum, EquipNum, RunFlag, FirstHVACIteration, InitLoopEquip, CurLoad, MaxLoad, MinLoad, OptLoad, GetCompSizFac, SizingFac, TempCondInDesign );
```

Comment:
Yeah, whenever we try to cleanup unused parameters (arguments), it's a balance between maintaining symmetry and cleaning up warnings.  Even as I started working on cleaning this up, I was very hesitant to remove some, but I'm more and more comfortable now.  If we want to add them back in to use them later, it's not hard to do.  In that case, we'd be going one-by-one into each Sim function and making modifications anyway.  So adding the argument back isn't hard.  

More likely, I'd say the Sim* function is going to be getting a nice "symmetric"/uniform signature when a large scale overhaul of this stuff gets done.  Something like: 

```int simulate( SimArguments & args );```

where ```SimArguments``` is a class/struct that will contain everything needed, and if we need to add another argument for simulate sometime, we just add it as a new variable on this class, and we don't have to modify hundreds of individual calls.  

So, for now, I'm much more in favor of cleaning up meaningless warnings even at the cost of symmetry, if it will help make the more important warnings visible.